app/controllers/course/dashboards_controller.rb

```diff
@@ -5,6 +5,5 @@ def show
     # TODO: make this not insane!
     @section = policy_scope(Chapter).first.sections.first
 
```

Comment:
So. I'll try and setup some actions to help with this, so it makes the PR process easier. But right now my rubocop accuses this:
![image](https://user-images.githubusercontent.com/1820441/103923367-a0744880-50da-11eb-98db-dbb7a4352c2b.png)

Also. Feel free to remove that comment. It's not insane anymore after your change. :grin:

Reply:
Sounds good. I will remove the comment and update the pull request.